.github/workflows/build_3.4.1_2.13.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: "Build and Test (3.4.1) for Scala 2.13"
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - '3.4.1/**'
+
+jobs:
+  run-build:
+    strategy:
+      matrix:
+        image-type: ["all", "python", "scala", "r"]
+    name: Run
+    secrets: inherit
+    uses: ./.github/workflows/main.yml
+    with:
+      spark: 3.4.1
+      scala: 2.13
```

Comment:
I think we can move `scala: 2.13` to strategy and squash this file to build_3.4.1.yaml:

```
    strategy:
      matrix:
        image-type: ["all", "python", "scala", "r"]
        scala: ["2.13", "2.12"]
```

Reply:
Should be fixed now.